ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -103,10 +108,19 @@ Item {
         }
     }
 
+    Connections {
+        target: root.messageStore
+
+        function onMessageMarkedAsUnreadChanged() {
+            d.setKeepUnread(true)
+        }
+    }
+
```

Comment:
I believe this is redundant to:
```
  proc markMessageAsUnread*(self: View, messageId: string) {.slot.} =
    self.delegate.markMessageAsUnread(messageId)
    self.setKeepUnread(true)
```
and can be removed from here.

Reply:
That's correct, thanks for catching that.